harness/helpers.ts

```diff
@@ -135,17 +136,64 @@ export const wait = (ms: number) => {
     })
 }
 
-export const callOnClientInitialized = async (clientId: string, url: string, callbackURL: string) => {
-    return await fetch(`${url}/client/${clientId}`, {
+export const callOnClientInitialized = async (
+    clientId: string, 
+    proxyUrl: string, 
```

Comment:
you could also obtain this one just by knowing the SDK name right? using `getConnectionStringForProxy` ?